src/penn_chime/charts.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+from typing import Optional
+
+from altair import Chart  # type: ignore
+import pandas as pd  # type: ignore
+import numpy as np  # type: ignore
+
+from .parameters import Parameters
+from .utils import add_date_column
+
+
+def new_admissions_chart(
+    alt,
+    projection_admits: pd.DataFrame,
+    parameters: Parameters,
+    as_date: bool = False,
+) -> Chart:
+    """docstring"""
+    plot_projection_days = parameters.n_days - 10
+    max_y_axis = parameters.max_y_axis
+
+    y_scale = alt.Scale()
+
+    if max_y_axis is not None:
+        y_scale.domain = (0, max_y_axis)
+        y_scale.clamp = True
+
+    tooltip_dict = {False: "day", True: "date:T"}
+    if as_date:
+        projection_admits = add_date_column(projection_admits)
+        x_kwargs = {"shorthand": "date:T", "title": "Date"}
+    else:
+        x_kwargs = {"shorthand": "day", "title": "Days from today"}
+
+    return (
+        alt.Chart(projection_admits.head(plot_projection_days))
+        .transform_fold(fold=["Hospitalized", "ICU", "Ventilated"])
+        .mark_line(point=True)
+        .encode(
+            x=alt.X(**x_kwargs),
+            y=alt.Y("value:Q", title="Daily admissions", scale=y_scale),
+            color="key:N",
+            tooltip=[
+                tooltip_dict[as_date],
+                alt.Tooltip("value:Q", format=".0f", title="Admissions"),
+                "key:N",
+            ],
+        )
+        .interactive()
+    )
+
+
+def admitted_patients_chart(
+    alt,
+    census: pd.DataFrame,
+    parameters: Parameters,
+    as_date: bool = False
+) -> Chart:
+    """docstring"""
+
+    plot_projection_days = parameters.n_days - 10
```

Comment:
Lint says that the variable plot_projection_days isn't used.

Reply:
good catch -- that was something i had commented out while debugging that had forgotten to uncomment.

Reply:
fixed.